src/master/develop/reading.md

```diff
@@ -156,8 +227,64 @@ try {
 :::
 
 ::: tab Python
-This feature is not yet supported or not documented.
-Do you want to make a feature request or help out? Open an issue on [Python sdk github project](https://github.com/codenotary/immudb-py/issues/new)
+
+Python immudb sdk currently doesn't support `VerifiedGetSince` method
```

Comment:
Please add an github issue for that

Reply:
https://github.com/codenotary/immudb-py/issues/39